src/cli/remove.rs

```diff
@@ -0,0 +1,87 @@
+use anyhow::{bail, Context, Result};
+use clap::Parser;
+use console::style;
+
+use rokit::{
+    discovery::discover_all_manifests, manifests::RokitManifest, storage::Home, tool::ToolAlias,
+};
+
+use crate::util::CliProgressTracker;
+
+/// Adds a new tool to Rokit and installs it.
+#[derive(Debug, Parser)]
+pub struct RemoveSubcommand {
+    /// The alias of the tool to remove.
+    pub alias: ToolAlias,
+    /// Remove this tool globally instead of adding
+    /// it to the nearest manifest file.
```

Comment:
```suggestion
    /// Remove this tool globally instead of removing
    /// it from the nearest manifest file.
```

Reply:
Oops, didn't edit the doc comments for when I copied the related functions to modify!